app/forms/gpo_verify_form.rb

```diff
@@ -76,7 +76,7 @@ def letter_count
     pending_profile&.gpo_confirmation_codes&.count
   end
 
-  def attempts
+  def submit_attempts
     RateLimiter.new(user: user, rate_limit_type: :verify_gpo_key).attempts
```

Comment:
I was tempted to rename the method for `RateLimiter` as well, as it seems to only be used by the methods I'm renaming here. But `RateLimiter` has a few of methods with `attempts` in the name, and I thought if I renamed one I would need to rename them all. That seemed out of scope for this work.

Reply:
I dont think we should change name in `RateLimiter`, from it's own perspective, it is `attempt`, it's for the clients to use it as different attempt.